cmd/provider/main.go

```diff
@@ -36,7 +36,8 @@ func main() {
 	var (
 		app            = kingpin.New(filepath.Base(os.Args[0]), "AWS support for Crossplane.").DefaultEnvars()
 		debug          = app.Flag("debug", "Run with debug logging.").Short('d').Bool()
-		syncPeriod     = app.Flag("sync", "Controller manager sync period duration such as 300ms, 1.5h or 2h45m").Short('s').Default("1h").Duration()
+		syncInterval   = app.Flag("sync", "Sync interval controls the minimum frequency at which all watched resources are reconciled.").Short('s').Default("1h").Duration()
+		pollInterval   = app.Flag("poll", "Poll interval controls how long after a successful reconcile an individual resource is reconciled.").Default("1m").Duration()
```

Comment:
Nit: This is an improvement, but it's still not super clear to the uninitiated reader when the minimum vs the other interval would come into play. Maybe an update to the poll interval to mention something about it being the goal or the soonest interval would help?

Reply:
@negz what would you think about something more abstract like "Controls how often an individual resource is checked for drift."? It feels a little less accurate, but it is more accessible to someone who isn't familiar with the underlying machinery. I could go either way.

Reply:
I think that sounds good for poll interval, and I'd maybe try frame sync interval as kind of a backup/fallback. Maybe:

* Poll: How often each individual resource should be checked for drift.
* Sync: How often all resources will be double checked for drift.

Reply:
Hi @hasheddan @negz, I looked at the updated help message for the `Sync` flag. As a newcomer to Crossplane or Kubernetes internals, the `double check` is still a little bit confusing as to why it is required. Looking up to controller-runtime docs helped clear the reasons for double-checking for drift. 

This change is definitely better than the previous message, and it is tough to sum up the official documentation in a one-line command, but can we add a reference to the official docs here? I feel it would help a lot to figure out the need for this sync interval. Thanks for the amazing work! :smile: